app/javascript/react/screens/App/Plan/components/PlanRequestDetailList/PlanRequestDetailList.js

```diff
@@ -68,14 +68,16 @@ class PlanRequestDetailList extends React.Component {
     pageChangeValue: 1
   };
 
-  componentDidUpdate() {
-    const {
-      failedMigrations,
-      successfulMigrations,
-      notificationsSentList,
-      dispatchVMTasksCompletionNotificationAction
-    } = this.props;
-    dispatchVMTasksCompletionNotificationAction(failedMigrations, successfulMigrations, notificationsSentList);
+  componentDidUpdate(prevProps, prevState) {
+    if (prevProps.planRequestTasks !== this.props.planRequestTasks) {
+      const {
```

Comment:
Unfortunately, it seems that this conditional is still firing every time `componentDidUpdate` gets called (I confirmed this by placing a breakpoint on line 79 and waiting for the next poll).  The reason is that `planRequestTasks` is a `seamless-immutable` array, and so, each time we poll the API, the new array generated is not equal to the previous in-memory array regardless of its contents.

One way to sidestep seamless is to use `JSON.stringify()`, and compare the deep cloned arrays.

Reply:
`componentDidUpdate` does get called each time, and that is OK.

I was just making sure that the notification toasts for VM Migration completed/failed don't appear each time I try to click on the download dropdown or sort the list (things that cause the component to update)

Reply:
ok good :)

I realized after I commented that my "fix" is particularly brittle b/c it relies on the order of the array elements

Reply:
The best way to see the toast notification issue that I am referring to above is -
- Work with the `master` branch
- Try and download a log

--> you will see that the toast notifications are getting triggered, when they shouldn't

Reply:
As we discussed, let's address this issue separately.

I have removed the commit from the PR.